app/views/admin/school_groups/show.html.erb

```diff
@@ -20,17 +23,37 @@
     <div class="card-deck">
       <div class="card">
         <div class="card-body">
-          <div>Active <span class="float-right badge badge-success"><%= @school_group.schools.visible.count %></span></div>
-          <div>Active (with data visible) <span class="float-right badge badge-success"><%= @school_group.schools.visible.data_enabled.count %></span></div>
-          <div>Invisible <span class="float-right badge badge-info"><%= @school_group.schools.not_visible.count %></span></div>
-          <div>Onboarding <span class="float-right badge badge-warning"><%= @school_group.school_onboardings.incomplete.count %></span></div>
-          <div>Removed <span class="float-right badge badge-secondary"><%= @school_group.schools.inactive.count %></span></div>
+          <div>Active
```

Comment:
Perhaps this ought to be changed to "Active (and visible)" now for clarity (given the contents of the active tab is now different). Alternative would be to add it as a tooltip?